include/xrpl/protocol/TxFlags.h

```diff
@@ -185,6 +204,18 @@ constexpr std::uint32_t tfDepositMask = ~(tfUniversal | tfDepositSubTx);
 constexpr std::uint32_t tfClearAccountCreateAmount     = 0x00010000;
 constexpr std::uint32_t tfBridgeModifyMask = ~(tfUniversal | tfClearAccountCreateAmount);
 
+// MPTokenIssuanceCreate flags:
+constexpr std::uint32_t const tfMPTokenIssuanceCreateMask  =
+  ~(tfMPTCanLock | tfMPTRequireAuth | tfMPTCanEscrow | tfMPTCanTrade | tfMPTCanTransfer | tfMPTCanClawback | tfUniversal);
+
+// MPTokenIssuanceDestroy flags:
+constexpr std::uint32_t const tfMPTokenIssuanceDestroyMask  = ~tfUniversal;
+
+// MPTokenAuthorize flags:
+constexpr std::uint32_t const tfMPTokenAuthorizeMask  = ~(tfMPTUnauthorize | tfUniversal);
+
+// MPTokenIssuanceSet flags:
+constexpr std::uint32_t const tfMPTokenIssuanceSetMask  = ~(tfMPTLock | tfMPTUnlock | tfUniversal);
```

Comment:
Please move the masks next to their flags, like the pattern for `AMM`, and please put `tfUniversal` first in the sequences.

Reply:
created a pr https://github.com/gregtatcam/rippled/pull/45/files